server/datastore/mysql/hosts.go

```
@@ -0,0 +1,43 @@
+package mysql
+
+import (
+	"time"
+
+	"github.com/kolide/kolide-ose/server/kolide"
+)
+
+func (d *Datastore) NewHost(host *kolide.Host) (h *kolide.Host, e error) {
```

Applies to the entire PR: please avoid named returns unless they add clarity. 
Use this as a guideline: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

---

you could always just `panic` in the body instead of `return` if you want the code to compile without named returns.